tests/resource_filtering_test.go

```diff
@@ -2409,7 +2412,7 @@ var _ = Describe("Resource excludes", func() {
 				By("Verifying included VM exists")
 				err = WaitForVirtualMachineStatus(*kvClient, namespace.Name, vmIncluded.Name, kvv1.VirtualMachineStatusStopped, kvv1.VirtualMachineStatusRunning)
 				Expect(err).ToNot(HaveOccurred())
-
+				time.Sleep(60 * time.Second)
```

Comment:
We really want to wait 60 seconds?

Reply:
no, that was a stop so I could do a troubleshooting

Reply:
removed